article/app/services/dotcomponents/RenderType.scala

```diff
@@ -1,7 +1,8 @@
 package services.dotcomponents
 
 sealed trait RenderType
-case object RemoteRender extends RenderType
-case object RemoteRenderAMP extends RenderType
-case object LocalRenderAmp extends RenderType
-case object LocalRenderArticle extends RenderType
+case object DotcomRendering extends RenderType
+case object DotcomRenderingAMP extends RenderType
+case object FrontendLegacyAMP extends RenderType
```

Comment:
Can we move this rename into a separate PR, please? This is a _monumental_ change and probably shouldn't be an aside but celebrated on it's own! 😎

Reply:
Done!